nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/AbstractEmailProcessor.java

```diff
@@ -348,14 +345,11 @@ private void transfer(Message emailMessage, ProcessContext context, ProcessSessi
         long start = System.nanoTime();
         FlowFile flowFile = processSession.create();
 
-        flowFile = processSession.append(flowFile, new OutputStreamCallback() {
-            @Override
-            public void process(final OutputStream out) throws IOException {
-                try {
-                    StreamUtils.copy(emailMessage.getInputStream(), out);
-                } catch (MessagingException e) {
-                    throw new IOException(e);
-                }
+        flowFile = processSession.append(flowFile, out -> {
```

Comment:
The email support was introduced prior to 1.0, so if this patch only meant for 1.0, then it's fine, but at the moment the JIRA is not versioned and I assume the fix should be applied on both branches 0.* and master